index.html

```diff
@@ -7,9 +7,7 @@
     <body>
         <h1>Welcome to Tyler's website</h1>
         <p>
-            This site was made entirely by Tyler and no one else. If anyone were to ever
-            make edits to this code, they should probably update this text because it
-            wouldn't be true anymore.
+            Hey Tyler the term is Shibboleth.
```

Comment:
Code suggestion: you should have told me this before the end of the presentation